layouts/shortcodes/eol-releases.html

```diff
@@ -0,0 +1,34 @@
+<h2 id="non-active-branch-history">Non-Active Branch history</h2>
+<p>
+    These releases are no longer supported.
+</p>
+<div>
+    <table class="table table-striped col-md-8">
+        <thead class="thead">
+            <tr>
+                <th class="col-md-2">Minor Version</th>
+                <th class="col-md-2">Final Patch Release</th>
+                <th class="col-md-2">EOL Date</th>
+                <th class="col-md-4">Note</th>
+            </tr>
+        </thead>
+        <tbody>
+            {{ range $eolRelease := .Site.Data.releases.eol.branches }}
+            <tr>
+                <td>
+                    {{ $eolRelease.release }}
+                </td>
+                <td>
+                    {{ $eolRelease.finalPatchRelease }}
+                </td>
+                <td>
+                    {{ $eolRelease.endOfLifeDate }}
+                </td>
+                <td>
+                    {{ $eolRelease.note }}
+                </td>
+            </tr>
+            {{ end }}
+        </tbody>
+    </table>
+</div>
```

Comment:
We can make this change, but we're then switching from something which is _easy_ to localize to something that is _hard_ to localize.

Reply:
Fair point. Are there guidelines or best practices that I can look at for localizing the terms used here?

Reply:
Have a look at https://github.com/kubernetes/website/pull/36375 for some recent similar work. You can also ask in #sig-docs-localizations on Slack for advice.